src/foam/dao/history/HistoryRecord.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao.history',
+  name: 'HistoryRecord',
+  documentation: `Contains an array of property updates`,
+  properties: [
+    {
+      class: 'FObjectProperty',
```

Comment:
Should just be a String. We don't want to store the whole User, which contains all of their contact information and can be quite large. Just their name. Maybe something like:
Greer, Kevin (kgr)
last-name, first-name (id)